app/helpers/urls.js

```diff
@@ -21,6 +21,12 @@ const getUrl = (urlObject, url, request, secBtn, _currentUrl) => {
     return  dependentAnswer != null ? thenUrl : elseUrl
   }
 
+  if (elseUrl === 'replacing-insulation' && 
```

Comment:
Same as in other PR - why do we need this if statement? can we not use nonDependentUrl instead?

Reply:
I had a look at nonDependentUrl and it looks like it might not be usable here (maybe), but there should be a way to rewrite this so it is generic and can be used for any page with a similar situation (as we have seen this once or twice in the past)